bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoCoffeeHandler.java

```diff
@@ -113,10 +112,9 @@ private void poll() {
                 logger.debug("Polling job");
 
                 // Check if the Wemo device is set in the UPnP service registry
-                // If not, set the thing state to ONLINE/CONFIG-PENDING and wait for the next poll
                 if (!isUpnpDeviceRegistered()) {
                     logger.debug("UPnP device {} not yet registered", getUDN());
-                    updateStatus(ThingStatus.ONLINE, ThingStatusDetail.CONFIGURATION_PENDING,
+                    updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE,
```

Comment:
Please check but I am almost sure your second parameter is just useless.

Reply:
> Please check but I am almost sure your second parameter is just useless.

I couldn't find any method overload taking only ThingStatus and description, so I believe it's needed to provide `ThingStatusDetail.NONE` in order to be able to provide a description.

Reply:
Oups, I did not see there was a third argument!